ui/src/main/java/org/jtaccuino/core/ui/spi/SheetManagerSPI.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 JTaccuino Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jtaccuino.core.ui.spi;
+
+import org.jtaccuino.core.ui.Sheet;
+
+public abstract class SheetManagerSPI {
```

Comment:
Any reasn for the abstract class and not simply an interface?

Reply:
Abstract class to now let multiple services be implemented by one class. This helps ServiceLoader usage.

Reply:
This is not a sealed class, so the number of classes implementing this service is not limited...

Reply:
That is assuming other SPI are not interface based